client/src/components/user-profile.jsx

```diff
@@ -12,10 +12,10 @@ function Proflie(props) {
     // console.log(props.user.interests);
     return (
         <div>
-            <img src={props.user.profilePicURL} />
-            <h1>@{props.user.username}</h1>
-            <h2>{props.user.name}</h2>
-            <ul>{props.user.interests.map(interest => <li>{interest}</li>)}</ul>
+            <img class="img-thumbnail" src={props.user.profilePicURL} />
```

Comment:
in React all html tags class needs to be className.
Also is img-thumbnail a class defined in bootstrap?